tests/services/nextflowtower/responses.py

```diff
@@ -527,3 +527,249 @@
     ],
     "totalSize": 3,
 }
+
```

Comment:
Out of curiosity, did you try and leverage copilot to generate this test response?

Reply:
I did not, I used an existing Hello World workflow run to get the example response. I will definitely consider doing that in the future!